projects/packages/assets/src/class-assets.php

```diff
@@ -450,9 +450,9 @@ public static function wp_default_scripts_hook( $wp_scripts ) {
 		$content_dir = Jetpack_Constants::get_constant( 'WP_CONTENT_DIR' );
 		$abspath     = Jetpack_Constants::get_constant( 'ABSPATH' );
 
-		if ( strpos( $lang_dir, $content_dir ) === 0 ) {
+		if ( \str_starts_with( $lang_dir, $content_dir ) ) {
```

Comment:
I'm not sure that `wp-includes/compat.php` winds up getting loaded from the `wp-admin/load-scripts.php` entry point where this hook gets called from.

Reply:
I note that `wp-admin/load-styles.php` has [a comment calling out that lack](https://github.com/WordPress/wordpress-develop/blob/0b8ca16ea3bd9722bd1a38f8ab68901506b1a0e7/src/wp-admin/load-styles.php#L76).

Reply:
That makes sense. c020b9f9e8ea93f414af7b966a86cb7ddd1715dc